### ISSUE TITLE
[Farbric 19.2] Seeds not compostable

### ISSUE THREAD

**Issue by Shibva** [MehVahdJukaar/Supplementaries#640]
The issue here is quite simple, Flax seeds for some reason are not compostable; guessing that part was forgotten or some part of the code broke

that is all, should overall be a quick fix

**Comment by MehVahdJukaar** [comment]
whats your mod version?

**Comment by Shibva** [comment]
It's been like this for a while so we're gone I'm surprise no one has reported it before me

**Comment by MehVahdJukaar** [comment]
I did try it yesterday and it was working for me

**Comment by Shibva** [comment]
Maybe it's because I'm using the quilt… Strange w why it's doing that thou

**Comment by Shibva** [comment]
Quilts API does it supposed to be very similar to fabrics to allow for backwards compatibility with fabric mods

if you need to do something different for it let me know, and I might bring it up with their api

**Comment by MehVahdJukaar** [comment]
As far as I know for compostable you just add stuff to the composter map directly. That's a vanilla thing I don't think there's any API for it and even if there was it should be compatible with the vanilla way. Bur that's likely it. Quilt claim that it works with fabric mod is their claim and could be a bit of a bold one as code is different so minor things might be different enough to cause discrepancies

**Comment by Shibva** [comment]
I can give it a try when I have a time; I just need to know what I need to do to add something to the composter through a data pack, if that's possible

Or you can try to fix it… To be honest, I'm not sure what your tears because I read your last Changelog, and you said that you don't wanna touch this sort of version ever again; yet again, I may be missing sarcasm, like I tend to do

**Comment by Shibva** [comment]
So I found a mod whose code is open-source and has compostable items that work both in fabric and quilt

https://github.com/0rc1nus/Galosphere-Main/blob/ff2716c789022ff133690e4deebbe8bbc597b636/src/main/java/net/orcinus/galosphere/init/GVanillaIntegration.java#L3
maybe this would tell what is being done differently for this issue to be fixed? tbh im curious at this point

**Comment by Xelbayria** [comment]
is the issue resolved? if yes, then please close. 
otherwise, you can leave it open.

**Comment by Shibva** [comment]
I don't believe so; it seems to be working fine on fabric, but not on quilt for whatever reason


It's been a while since I've checked, since I moved off of quilt due to some issues that I've ran into. It's highly likely that the issue is still present

**Comment by Shibva** [comment]
It's ironic; it should, but it's not really.
I will try to see any quilt developers would know what's going on; they might have an idea what's causing this bug.

Thanks for your feedback

**Comment by meeprdib** [comment]
Mod versions on 1.19.2 and below are not supported
